extensions/wikia/FlowTracking/scripts/createPageTracking.js

```diff
@@ -45,7 +70,22 @@ define(
 			return namespaceId === 0;
 		}
 
+		function isTitleInMainNamespace(title) {
+			var namespace = '';
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this useless assignment to local variable "namespace" [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=javascript%3AS1854)